File: loops/fizzbuzz.c
// James Wilcox, FizzBuzz C
#include <stdio.h>
#include <string.h>

int num = 1;

int main(void){
while (num <= 50) {  
    if (num % 3 == 0 && num % 5 == 0) {  
        printf("FizzBuzz\n");
    } else if (num % 3 == 0) {  
        printf("Fizz\n");
    } else if (num % 5 == 0) {  
        printf("Buzz\n") ;
    } else {  
        printf("%d\n", num);
    }  
    num+=1;  
}  

    return 0;
}

File: programming/loops/fizzbuzz.py
# James Wilcox, FizzBuzz Python

num = 1
while num <=50:
    if (num % 3 == 0) & (num % 5 == 0):
        print("FizzBuzz")
        num +=1
    elif (num % 3 == 0):
        print("Fizz")
        num +=1
    elif (num % 5 == 0):
        print("Buzz")
        num +=1
    else:
        print(num)
        num +=1